beacon-chain/rpc/validator/sync_committee.go

```diff
@@ -47,14 +53,25 @@ func (vs *Server) SubmitSyncMessage(ctx context.Context, msg *ethpb.SyncCommitte
 
 // GetSyncSubcommitteeIndex is called by a sync committee participant to get its subcommittee index for sync message aggregation duty.
 func (vs *Server) GetSyncSubcommitteeIndex(ctx context.Context, req *ethpb.SyncSubcommitteeIndexRequest) (*ethpb.SyncSubcommitteeIndexRespond, error) {
+	indices, err := vs.syncSubcommitteeIndex(ctx, bytesutil.ToBytes48(req.PublicKey), req.Slot)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Could not get sync subcommittee index: %v", err)
+	}
+	return &ethpb.SyncSubcommitteeIndexRespond{
+		Indices: indices,
+	}, nil
+}
+
+// syncSubcommitteeIndex returns a list of subcommittee index of a validator and slot for sync message aggregation duty.
+func (vs *Server) syncSubcommitteeIndex(ctx context.Context, pubkey [48]byte, slot types.Slot) ([]uint64, error) {
 	headState, err := vs.HeadFetcher.HeadState(ctx)
```

Comment:
There is a much more efficient way to use retrieve this(ex: the sync committe subnet cache). However that is in the other PR, which might take a bit longer to get in. We can improve on this later.